src/assets/wise5/services/teacherProjectService.ts

```diff
@@ -344,7 +344,7 @@ export class TeacherProjectService extends ProjectService {
    * @param node the new node
    * @param nodeId the node to add after
    */
-  createNodeAfter(newNode, nodeId) {
+  createNodeAfter(newNode: any, nodeId: string) {
```

Comment:
Also add return type?
```suggestion
  createNodeAfter(newNode: any, nodeId: string): void {
```